common/supplemental/coverageLevels.xml

```diff
@@ -92,7 +92,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageVariable key="%keys80" value="(calendar|cf|collation|currency|em|fw|hc|lb|lw|ms|rg|ss|numbers|d0|h0|i0|k0|m0|s0|t0|x0)"/>
 		<coverageVariable key="%keys100" value="(col(Alternate|Backwards|CaseFirst|CaseLevel|HiraganaQuaternary|Normalization|Numeric|Reorder|Strength)|kv|sd|timezone|va|variableTop|x)"/>
 		<coverageVariable key="%language30" value="und"/>
-		<coverageVariable key="%language40" value="(de(_(AT|CH))?|en(_(AU|CA|GB|US))?|es(_(ES|419|MX))?|fr(_(CA|CH))?|it|ja|pt(_(BR|PT))?|ru|zh(_(Hans|Hant))?)"/>
+		<coverageVariable key="%language40" value="(de(_(AT|CH))?|en(_(AU|CA|GB|US))?|es(_(ES|419|MX))?|fr(_(CA|CH))?|it|ja|kgp(_(BR))?|pt(_(BR|PT))?|ru|yrl(_(BR|CO|VE))?|zh(_(Hans|Hant))?)"/>
```

Comment:
This is not correct: we only translate a small number of languages at level 40. It should go to modern instead.

Reply:
Perfect, Mark. As I've mentioned in the CR, I've separated the PRs exactly to confirm the exact info. 
I'll move to modern (level 80) as per TR35.
Thanks for confirming.

Reply:
Mark, one point here is that moving to modern, the test TestLanguageNameCoverage will fail because we still don't have localized language names for those languages. Based on that, what would it be the recommendation? Proceed with the change?
One note here: to deal with that on Android, internally we've gathered the language names with native translators to avoid display the language code, but of course, it does contemplate a subset of CLDR locales. 

Please let me know your thoughts.

Reply:
I think we need the language names in English, but not the other languages. But I'm kinda swamped this week, so I won't be able to spend any time on answering your (very sensible) questions.

You can ping me Monday by email, if that's ok.

Reply:
Noted. I can send an email next Monday. tks